extended/src/main/java/apoc/vectordb/ChromaDb.java

```diff
@@ -0,0 +1,220 @@
+
+        VectorEmbeddingConfig apiConfig = CHROMA.get().getEmbedding().fromGet(config, procedureCallContext, getStringIds(ids));
+        return executeRequest(apiConfig.getApiConfig(), urlAccessChecker)
+                .map(v -> (List) v)
+                .map(ListResult::new);
+    }
+
+    @Procedure(value = "apoc.vectordb.chroma.get", mode = Mode.SCHEMA)
+    @Description("apoc.vectordb.chroma.get(hostOrKey, collection, ids, $configuration) - Get the vectors with the specified `ids`")
+    public Stream<EmbeddingResult> query(@Name("hostOrKey") String hostOrKey,
+                                                      @Name("collection") String collection,
+                                                      @Name("ids") List<Object> ids,
+                                                      @Name(value = "configuration", defaultValue = "{}") Map<String, Object> configuration) throws Exception {
+        String url = "%s/api/v1/collections/%s/get";
+        Map<String, Object> config = getVectorDbInfo(hostOrKey, collection, configuration, url);
+
+        VectorEmbeddingConfig apiConfig = CHROMA.get().getEmbedding().fromGet(config, procedureCallContext, ids);
+        return getEmbeddingResultStream(apiConfig, procedureCallContext, urlAccessChecker, tx,
+                v -> listToMap((Map) v).stream());
+    }
+
+    @Procedure(value = "apoc.vectordb.chroma.query", mode = Mode.SCHEMA)
+    @Description("apoc.vectordb.chroma.query(hostOrKey, collection, vector, filter, limit, $configuration) - Retrieve closest vectors the the defined `vector`, `limit` of results,  in the collection with the name specified in the 2nd parameter")
+    public Stream<EmbeddingResult> query(@Name("hostOrKey") String hostOrKey,
+                                                      @Name("collection") String collection,
+                                                      @Name(value = "vector", defaultValue = "[]") List<Double> vector,
+                                                      @Name(value = "filter", defaultValue = "{}") Map<String, Object> filter,
+                                                      @Name(value = "limit", defaultValue = "10") long limit,
+                                                      @Name(value = "configuration", defaultValue = "{}") Map<String, Object> configuration) throws Exception {
+
+        String url = "%s/api/v1/collections/%s/query";
+        Map<String, Object> config = getVectorDbInfo(hostOrKey, collection, configuration, url);
+
+        VectorEmbeddingConfig apiConfig = CHROMA.get().getEmbedding().fromQuery(config, procedureCallContext, vector, filter, limit, collection);
+        return getEmbeddingResultStream(apiConfig, procedureCallContext, urlAccessChecker, tx,
+                v -> listOfListsToMap((Map) v).stream());
+    }
+
+    private Map<String, Object> getVectorDbInfo(String hostOrKey, String collection, Map<String, Object> configuration, String templateUrl) {
```

Comment:
it would be great if we could expose this `getVectorDbInfo` in a procedure call for each of the databases to get an overview what's in there.

Reply:
Sorry, what I meant was not the configuration stored on the neo4j side, but rather the metadata (which collections and sizes are available in the vector db) but we can also do that post 5.20